src/contexts/useAuth.jsx

```diff
@@ -1,42 +1,40 @@
-import React, {createContext, useContext, useEffect, useState} from 'react'
-import {auth} from '../auth/firebase.jsx'
-import {useUser} from "./useUser";
+import React, { createContext, useContext, useEffect, useState } from 'react';
```

Comment:
Just realized that useAuth should not be a jsx file 
It should be useAuth.js

Reply:
Will fix that.